src/Uno.UI.Composition/Composition/Visual.cs

```diff
@@ -133,6 +133,7 @@ public Vector3 RotationAxis
 		private protected override void OnPropertyChangedCore(string? propertyName, bool isSubPropertyChange)
 		{
 			Compositor.InvalidateRender(this);
+			InvalidatePaint(); // TODO: only repaint when "dependent" properties are changed
```

Comment:
Thinking about this (and the default private `_requiresRepaint = true;`), I realized that actually most of our `Visual` in uno apps will never have anything to draw, and we will create an incredible amount of empty `_picture` that are blank (not even considering the cost of the recording canvas etc.).